src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -112,11 +118,25 @@ public void setClient(Client target, Client editedClient) {
         addressBook.setClient(target, editedClient);
     }
 
+    @Override
+    public boolean hasCountryNote(Country country, Note countryNote) {
+        requireAllNonNull(country, countryNote);
+
```

Comment:
This whitespace is little bit weird lol, but might just be my personal preference

Reply:
I checked and its still 4 spaces lol